lib/provider/boku.py

```diff
@@ -33,6 +33,10 @@ def create(self, request):
         cleaned = form.cleaned_data
         transaction = cleaned['param']
 
+        # Check the signature shows it was from Boku.
+        log.info('Checking signature')
+        check_sig(request.DATA)
```

Comment:
I don't see a test that executes this view. I think it would be nice to have a test like that, at least something with a mock of ``check_sig`` to assert that it was called.